students/15th/teamteam/hyeonju/westagram/user/views.py

```diff
@@ -0,0 +1,72 @@
+#from django.shortcuts import render
+
+import json
+import re
+import bcrypt
+import jwt
+
+from django.views import View
+from django.http import JsonResponse
+from .models import Users
+from my_settings import SECRET
+
+# Create your views here.
+class SignupView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        
+        email_validation = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+        pw_validation = '^[A-Za-z0-9@#$%^&+=]{8,}$'
+
+        try:
+
+            if not re.match(email_validation, data['email']):
+                return JsonResponse(
+                    {"message":"INVALID_ID"}, status = 401
+                    )
+
+            if not re.match(pw_validation, data['password']):
+                return JsonResponse(
+                    {"message":"INVALID_PW"}, status = 400
+                    )
+
+
+            if Users.objects.filter(email=data['email']).exists():
+                return JsonResponse(
+                    {"message":"USER_EXIST"}, status = 400
+                    )
+
+            else:
+               
+                Users.objects.create(
+                    email = data['email'],
+                    password = bcrypt.hashpw(data['password'].encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
+                    ).save()
+
+                return JsonResponse({'message': 'SUCCESS'}, status = 200)
```

Comment:
자원이 생성되었을 때 상태 코드는 201를 사용합니다.
```suggestion
                return JsonResponse({'message': 'SUCCESS'}, status = 201)
```